src/validation/tests/assignment_validation_tests.rs

```diff
@@ -733,6 +732,49 @@ fn invalid_action_call_assignments_are_validated() {
     assert_validation_snapshot!(&diagnostics)
 }
 
+#[test]
+fn action_call_parameters_are_only_validated_outside_of_parent_pou_contexts() {
+    let diagnostics = parse_and_validate(
```

Comment:
Can you use the buffered validator here?

Reply:
Agree the buffered validator makes the error messages here much easier